master/internal/api_experiment.go

```diff
@@ -499,6 +502,7 @@ func (a *apiServer) PatchExperiment(
 		"patch_experiment",
 		req.Experiment.Id,
 		marshalledPatches,
+		exp.Notes,
```

Comment:
rewriting the notes every time is a bit disappointing but i don't think it'll cause issues.